Enigma case study

m1073496 opened the pull request:
First pass at Enigma Case Study page. 

<img width="1791" alt="Screen Shot 2022-01-26 at 11 34 32 AM" src="https://user-images.githubusercontent.com/73088654/151225330-d8831712-a2a5-434b-a41c-b49ecfe88812.png">

<img width="1786" alt="Screen Shot 2022-01-27 at 1 22 04 PM" src="https://user-images.githubusercontent.com/73088654/151437614-842f902a-0542-4076-96e0-b36ce5a6d769.png">

<img width="1789" alt="Screen Shot 2022-01-27 at 1 22 12 PM" src="https://user-images.githubusercontent.com/73088654/151437620-0e265df1-2674-44d3-a9b9-a769db5d996b.png">

<img width="1792" alt="Screen Shot 2022-01-27 at 1 23 28 PM" src="https://user-images.githubusercontent.com/73088654/151437720-8e975890-a8ea-421e-beb9-27922d694628.png">

hannakim91 commented on pages/work/enigma.vue:
Some rewording ideas:

In order to successfully launch this application, the Enigma and Ship Shape development teams focused on implementing the following key features that make Enigma a unique assessment tool that meets its niche clients' needs.

hannakim91 commented on pages/work/enigma.vue:
These features [are] included the following crucial workflows:

hannakim91 commented on pages/work/enigma.vue:
to [securely] access their respective accounts

hannakim91 commented on pages/work/enigma.vue:
~various~

(I think this is sometimes a filler word)

hannakim91 commented on pages/work/enigma.vue:
Is creating different types of questions a unique feature too (MC/blank line/etc)? or too specific for this list maybe?

m1073496 replied:
Hmm, I get where you're coming from, although I don't know if that fits here since the employer doesn't have the ability to create different types of questions. May leave off.

hannakim91 commented on pages/work/enigma.vue:
Wondering if "and candidates to take" might be redundant since why would it only be half a feature (assign a test that can't be taken)?

hannakim91 commented on pages/work/enigma.vue:
Ship Shape’s main priority was to successfully build and launch a feature rich application~,~ according to the client’s schedule, and we’re happy to report that we met this target.

^comma unnecessary?
and would it be unprofessional to do a "!" instead of "." at the end of this sentence?

m1073496 replied:
I say we go for it!

hannakim91 reviewed (commented):
Awesome! Think you flow through all the necessary parts of the case study with just the right amount of detail. Mostly suggestions on sentence structure, if you like.

rwwagner90 commented:
@m1073496 I think the images should be wider to match the format of other case studies. The images and the background with the logo should be the same width and the text content is less wide, I believe. It also should alternate colors between brand color and that gray color. Does Enigma have a brand primary color or is it just black?

m1073496 commented:
> @m1073496 I think the images should be wider to match the format of other case studies. The images and the background with the logo should be the same width and the text content is less wide, I believe. It also should alternate colors between brand color and that gray color. Does Enigma have a brand primary color or is it just black?

@rwwagner90, I pulled the dark blue color from enigma's theme and updated the background color on the first image, and adjusted the size on the rest! Thanks!

rwwagner90 commented on pages/work/enigma.vue:
```suggestion
      <div class="bg-enigma-blue flex justify-center lg:flex-1">
```

rwwagner90 replied:
These boxes should alternate between brand color and grey.

rwwagner90 commented on pages/work/enigma.vue:
Looks like a duplicate alt here. Can we make these more descriptive?

rwwagner90 commented on pages/work/enigma.vue:
```suggestion
        <div class="pl-4 pr-4 pt-4 lg:pl-24 lg:pr-24 lg:pt-12">
```

I think to match the format of the other case studies, image containers should have these classes?

m1073496 replied:
Oops, you're right. I accidentally looked at other case studies' SVG containers as a reference. Reverting now.

rwwagner90 commented:
Thanks @m1073496 we're getting close! Left some more feedback.

rwwagner90 reviewed (changes requested):
Just a few potential changes here, but overall looking very good!

m1073496 commented:
> Just a few potential changes here, but overall looking very good!

Got it!
